### PR TITLE
Major borer update/changes

### PR DESCRIPTION
**Pull request by ShadowDeath6**
no issue :^)
### Intent of your Pull Request

Borers, in my opinion, had to rely too much on their host accepting them, and if the host DIDN'T accept them, they were doomed as soon as someone who knew how to do surgery came along. Hopefully, this makes the "dangerous parasite" feel of borers a bit more prominent.
### Changes

Borers now gain chemicals twice as fast.
Borers can gain chemicals while their host is unconscious or dead.
Borers can infest the dead, but not assume control to prevent them bugging out as ghosts.
Borers can infest braindeads, but not catatonics.
Borers have no cooldown on assuming control.

**INFLUENCE IS COMMENTED OUT, IT NO LONGER TAKES EFFECT**, meaning...
Borers can assume control immediately upon entering a person.
Borers are not restricted by influence to use certain chemicals or punishments.

Also, some misc changes.

Adds epinephrine to stabilize crit, and removes "stabilizes crit" from ephedrine, as it did not actually do it.
Removes colorful reagent as it was in the code but never used, and wasn't realistic.
Fixes jumpstart host fully healing the host regardless if they're dead or not, it now only works when they're dead.
Borers can now attack (click) a mob to scan its health and chemical contents, this includes scanning their host while inside them.
#### Changelog

:cl: ShadowDeath6
rscdel: Borers no longer use influence.
rscadd: Borers can attack(click) a mob to scan its health and chemical contents.
tweak: Borers can now infest braindeads, but not catatonics.
bugfix: Borers can no longer use Jumpstart Host when their host is alive.
bugfix: Borers can no longer assume control of a dead host.
tweak: Borers generate chemicals twice as fast.
rscadd: Borers have evolved the ability to create epinephrine, which will allow them to stabilize critical hosts.
rscadd: Borers can now infest dead bodies.
tweak: Borers now generate chemicals while their host is unconscious or dead.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
I need to sleep so I'll get to the comments tomorrow

**Comment by oisin100** [comment]
> rscdel: Borers no longer use influence.

I was doing this in #1037 

> rscadd: Borers can attack(click) a mob to scan its health and chemical contents.

Same goes here, But i like your way better.

> bugfix: Borers can no longer assume control of a dead host.

Thats intended.

> tweak: Borers generate chemicals twice as fast.

I was also working on something alike in #1037

**Comment by Time-Green** [comment]
Can borers be made aghost friendly?

**Comment by ShadowDeath6** [comment]
I thought you weren't working on it since you were busy with headcoder duties @oisin100 

As for assuming control of a dead host, it causes a weird bug where the borer can ghost from the body while still being alive, which was really janky and served no actual purpose from what I can see.

**Comment by ghost** [comment]
:-1: on braindead infestation

**Comment by ShadowDeath6** [comment]
> :-1: on braindead infestation

Why? It's not like they're infesting monkey-humans, and braindeads let borers get more hosts.

**Comment by ghost** [comment]
> permanent pilot over a host at will because they're braindead

no

**Comment by ShadowDeath6** [comment]
Permanent control still requires they release to see hivemind, borer HUD, and give the host healing chemicals, not to mention putting a big "infested" medHUD sign over their head.

**Comment by X-TheDark** [comment]
Funnily enough, right now Borers generate chems if the host is Alive/Dead, but not while he's unconscious.

**Comment by oisin100** [comment]
Finishing my PR will take some time, Ill merge this and then write over Shadows code when im done, Unless anyone else has any more input.

Shadow puts up a good point about permanent control.

**Comment by ShadowDeath6** [comment]
merg dis

File: code/modules/mob/living/simple_animal/borer/borer_captive.dm [hunk 1]
```diff
@@ -42,10 +42,10 @@
 		var/mob/living/simple_animal/borer/B = src.loc
 		var/mob/living/captive_brain/H = src
 
-		H << "<span class='danger'>You begin doggedly resisting the parasite's control (this will take approximately [B.influence-50] seconds).</span>"
+		H << "<span class='danger'>You begin doggedly resisting the parasite's control (this will take approximately 10 seconds).</span>"
 		B.victim << "<span class='danger'>You feel the captive mind of [src] begin to resist your control.</span>"
 
-		spawn(max(10, (B.influence-50)*10))
+		spawn(max(10, (100)))
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_captive.dm]: ....

Just change this to spawn(100)

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm [hunk 2]
```diff
@@ -249,19 +249,23 @@
...
 		return
 
-	if(world.time - used_control < control_cooldown)
-		src << "<span class='warning'>Its too soon to use that again!</span>"
+	if(victim.stat == DEAD)
+		src << "<span class='warning'>This host lacks enough brain function to control.</span>"
 		return
 
-	if(influence < 50)
+/*	if(world.time - used_control < control_cooldown)
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm]: Dont want spam control, keep this.

**Comment by ShadowDeath6** [comment]
so my program thing is messing up and can't load anything that isn't up to date with my most recent stuff, I can't make any more changes

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm (hunk 2 above)
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm]: we really shouldn't force borers to be docile, having a cooldown makes them weaker

**Comment by ShadowDeath6** [comment]
ugh hold on

File: code/modules/mob/living/simple_animal/borer/borer.dm [hunk 3]
```diff
@@ -173,7 +177,7 @@ var/total_borer_hosts_needed = 10
 		src << "<span class='warning'>[victim] is already infected!</span>"
 		return
 
-	if(!victim.key || !victim.mind || !victim.client)
```
Comment (AsV9) [inline code/modules/mob/living/simple_animal/borer/borer.dm:176]: > removing `if(!victim.client)`

:-1:

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm (hunk 2 above)
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm]: Not having a cooldown makes them annoying;.

**Comment by ShadowDeath6** [comment]
I can't edit my stuff so you guys'll have to take it like this, which works fine, or close it and wait for oisin to finish his thing

**Comment by oisin100** [comment]
> I can't edit my stuff so you guys'll have to take it like this, which works fine, or close it and wait for oisin to finish his thing

I need those things I asked to be un-reverted.

File: code/modules/mob/living/simple_animal/borer/borer.dm (hunk 3 above)
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer.dm:176]: ree I already went over this

**Comment by ShadowDeath6** [comment]
wew

**Comment by ShadowDeath6** [comment]
@oisin100 
please merge I did everything you asked

**Pull request merged**